packages/vite-plugin-vue-i18n/src/index.ts

```diff
@@ -210,7 +210,9 @@ function pluginI18n(
     async handleHotUpdate({ file, server }) {
       if (/\.(json5?|ya?ml)$/.test(file)) {
         const module = server.moduleGraph.getModuleById(
-          INTLIFY_BUNDLE_IMPORT_ID
+          getVirtualId(
+            INTLIFY_BUNDLE_IMPORT_ID
+          )
```

Comment:
```suggestion
          asVirtualId(INTLIFY_BUNDLE_IMPORT_ID)
```

getVirtualId removes the leading `\0`, possibly returning null. In this case we want to add `\0` (see @ghostiam’s comment at https://github.com/intlify/bundle-tools/issues/153#issuecomment-1209653696, `server.moduleGraph.getModuleById` only works with a leading `\0`).

I have tested this locally, and live reloading works when using `asVirtualId`.

Reply:
mb, I have remembered the name of incorrect function.